app/models/concerns/language_and_schedule_references.rb

```diff
@@ -5,6 +5,18 @@ module LanguageAndScheduleReferences
     has_many :language_skills, as: :languageable, dependent: :destroy
     accepts_nested_attributes_for :language_skills, allow_destroy: true
 
+    def native_language
+      return if language_skills.blank? || language_skills.native.blank?
+      language_skills.native.german.first || language_skills.native.first || LanguageSkill.new
+    end
+
+    def other_languages
+      return language_skills if language_skills.blank? || language_skills.native.blank?
+      language_skills.reject do |lang|
```

Comment:
Its not non german, its without the one that is considered native_lanugage. That could well be any other language but german. I prefer german, because I expect to have bilingual people, that have german and some other language as native_spoken in there set. In that case I'd prefer to show german, because it could be misleading otherwise.